pombola/writeinpublic/views.py

```diff
@@ -55,11 +55,41 @@ def object_ids(self, objects):
         return [p.everypolitician_uuid for p in objects]
 
 
+class CommitteeAdapter(object):
+    def filter(self, ids):
+        return Organisation.objects.filter(id__in=ids)
+
+    def get(self, object_id):
+        return Organisation.objects.get(pk=object_id)
+
+    def get_by_id(self, object_id):
+        return self.get(object_id)
+
+    def get_form_kwargs(self):
+        return {
+            'queryset': Organisation.objects.filter(
+                kind__name='National Assembly Committees',
+                contacts__kind__slug='email'
+            )
+        }
+
+    def object_ids(self, objects):
+        return [org.id for org in objects]
+
+
 class WriteInPublicMixin(object):
     def dispatch(self, *args, **kwargs):
-        configuration = Configuration.objects.get(slug=kwargs['configuration_slug'])
-
-        self.adapter = PersonAdapter()
+        configuration_slug = kwargs['configuration_slug']
+        configuration = Configuration.objects.get(slug=configuration_slug)
+
+        # FIXME: It would be nice if we didn't hardcode the configuration_slug
+        # values here.
+        if configuration_slug == 'south-africa-assembly':
+            self.adapter = PersonAdapter()
+        elif configuration_slug == 'south-africa-committees':
+            self.adapter = CommitteeAdapter()
+        else:
+            raise Exception, "Unknown configuration_slug: {}".format(configuration_slug)
```

Comment:
There's a minor PEP 8 thing here - it recommends using `raise Exception('message')` instead of `raise Exception, 'message'`. However, I think I'd write this differently anyway, more like:
```python
self.adapter = {
    'south-africa-assembly': PersonAdapter,
    'south-africa-committees': CommitteeAdapter,
}[configuration_slug]()
```